### ISSUE TITLE
Wrong french intword values

### ISSUE THREAD

**Issue by dingens** [jmoiron/humanize#25]
Using `fr_FR`, as of 10**12 (english "trillion", french "billion"), the `intword` function returns wrong results. See https://en.wikipedia.org/wiki/Long_and_short_scales

```
>>> humanize.i18n.activate('fr_FR')
<gettext.GNUTranslations instance at 0x7f7eeb455e60>
>>> humanize.intword(10**6)
'1.0 million'
>>> humanize.intword(10**9)
'1.0 milliard'
>>> humanize.intword(10**12)
'1.0 trillion'
>>> humanize.intword(10**15)
'1.0 quadrillion'
```

**Comment by johnyf** [comment]
A related issue is adding a note about the default scale (w/o locale activation).

**Comment by hugovk** [comment]
Will be fixed by https://github.com/jmoiron/humanize/pull/70.

**Comment by hugovk** [comment]
Thanks for the report, fixed by #70.
